build/common/installer/scripts/fluent-bit-conf-customizer.rb

```diff
@@ -2,6 +2,7 @@
 require_relative "ConfigParseErrorLogger"
 
 @fluent_bit_conf_path = "/etc/opt/microsoft/docker-cimprov/fluent-bit.conf"
+@fluent_bit_common_conf_path = "/etc/opt/microsoft/docker-cimprov/fluent-bit-common.conf"
```

Comment:
we need to do same the for fluent-bit-common.conf file in windows also.

Reply:
@ganga1980 thanks for pointing it out. added it below.